pkg/kubelet/qos/policy.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package qos
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+)
+
+// The OOM score of a process is the percentage of memory it consumes multiplied by 100
+// (barring exceptional cases). Giving over a 1000 difference between top tier and best effort
+// pods means that best-effort pods will be killed before top-tier pods.
+const (
+	PodInfraOomAdj   int = -501
```

Comment:
I'm making it -701. I don't want to make it lower, because the kubelet/kube-proxy are more critical, so I need to reserve a lower oom adjust score for those critical tasks. What do you think?